internal/home/upgrade.go

```diff
@@ -720,7 +719,7 @@ func upgradeSchema12to13(diskConf yobj) (err error) {
 	var dns yobj
 	dns, ok = dnsVal.(yobj)
 	if !ok {
-		return fmt.Errorf("unexpected type of dns: %T", dnsVal)
+		return fmt.Errorf("unexpected type of dns: %T; want: %T", dnsVal, dns)
```

Comment:
There are a few other similar errors throughout the recent migrations, and it would be nice to have them all in a unified format.

Reply:
I had added this to help me understand better what the code expected, but I agree it is more a debug printf.

I am not sure I understand your comment thou: do you want me to revert this change or to extend it to all the similar "unexpected ..." ?

Reply:
I mean that it's better to replace *all* instances of errors like `unexpected type of %s: %T` with this new form that includes the expected type as opposed to just here.